test/server/cardImplementations/01_SOR/MonMothmaVoiceoftheRebellion.spec.js

```diff
@@ -0,0 +1,126 @@
+                expect(this.cellBlockGuard).toBeInBottomOfDeck(this.player1, 4);
+                expect(this.pykeSentinel).toBeInBottomOfDeck(this.player1, 4);
+                expect(this.volunteerSoldier).toBeInBottomOfDeck(this.player1, 4);
+            });
+
+            it('should be allowed to choose nothing and place all cards on the bottom of the deck', function() {
+                this.player1.clickCard(this.monMothma);
+                this.player1.clickPrompt('Take nothing');
+
+                // Ensure that cards have moved to bottom of deck
+                expect(this.battlefieldMarine).toBeInBottomOfDeck(this.player1, 5);
+                expect(this.cartelSpacer).toBeInBottomOfDeck(this.player1, 5);
+                expect(this.cellBlockGuard).toBeInBottomOfDeck(this.player1, 5);
+                expect(this.pykeSentinel).toBeInBottomOfDeck(this.player1, 5);
+                expect(this.volunteerSoldier).toBeInBottomOfDeck(this.player1, 5);
+            });
+
+            it('should allow selection when deck has less than five cards', function() {
+                this.player1.setDeck([this.battlefieldMarine, this.cellBlockGuard, this.cartelSpacer]);
+                this.player1.clickCard(this.monMothma);
+                expect(this.player1).toHavePromptButtons([this.battlefieldMarine.title, 'Take nothing']);
+                expect(this.player1).toHaveDisabledPromptButtons([this.cartelSpacer.title, this.cellBlockGuard.title]);
+                this.player1.clickPrompt(this.battlefieldMarine.title);
+
+                // Ensure that cards have moved to bottom of deck
+                expect(this.player1.deck.length).toBe(2);
+                expect(this.cartelSpacer).toBeInBottomOfDeck(this.player1, 2);
+                expect(this.cellBlockGuard).toBeInBottomOfDeck(this.player1, 2);
+            });
+
+            it('when the deck is empty', function() {
+                this.player1.setDeck([]);
+                expect(this.player1.deck.length).toBe(0);
+
+                // Check results
+                this.player1.clickCard(this.monMothma);
+                expect(this.monMothma.location).toBe('ground arena');
+                expect(this.player1.deck.length).toBe(0);
+
+                // Player 2 now active
+                expect(this.player2).toBeActivePlayer();
+            });
+        });
+
+        describe('Mon Mothma\'s Ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mon-mothma#voice-of-the-rebellion'],
+                        deck: ['cell-block-guard', 'pyke-sentinel', 'volunteer-soldier', 'cartel-spacer', 'academy-defense-walker', 'wampa', 'viper-probe-droid', 'snowtrooper-lieutenant'],
+                        deckSize: 8,
+                        leader: ['leia-organa#alliance-general'],
```

Comment:
I changed things so that leader and base can no longer be a list, must be a single value. But the default resource count is now 20 so probably don't need to worry about aspect cost, can just remove the leader completely in almost all cases and use the default.